README.md

```
@@ -1,3 +1,134 @@
+## :rocket: 게임 시작
+
+- [x] 게임시작 문구 출력
+
+```
+장기 게임을 시작하겠습니다!
+```
+
+## :rocket: 기물 배치
+
+- [x] 초와 한의 초기 배치를 입력받을 수 있다.
+    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)